inc/namespace.php

```diff
@@ -130,26 +141,59 @@ function get_config() {
 }
 
 /**
- * Load the Object Cache dropin.
+ * Load the object cache.
+ *
+ * Check the object caching configuration and load either memcached
+ * or redis as appropriate.
+ *
+ * @deprecated 1.1.0 Object caching setup moved to dedicated functions.
  */
 function load_object_cache() {
-	wp_using_ext_object_cache( true );
+	if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
+		trigger_error(
+			sprintf(
+				__( '%1$s is <strong>deprecated</strong> since version %2$s! Use %3$s instead.' ),
```

Comment:
This is a core translation string so I've not put the `altis` domain in.

I am not sure whether the i18n are available at this point so it may need to be removed.

Reply:
@peterwilsoncc good call checking on that - the translation functions aren't available at that time, it's fine for this to be an untranslated string as it's not user facing.

Reply:
```suggestion
				'%1$s is deprecated since version %2$s! Use %3$s instead.',
```